### PR TITLE
Ready use for google maps

### LINKED ISSUE #258: Google map doesn't work
Can you please help me with that? I was not able to make it work even though it should be only about adding one key into `AndroidManifest.xml`...

### PR DESCRIPTION
**Pull request by tbergq**
closes #258

### CODE REVIEW

**Comment by tbergq** [comment]
You need to use an Emulator with API version 24

**Comment by mrtnzlml** [comment]
@tbergq Always exactly 24? This should be in the docs. What about the `yarn android`?

File: android/.project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mrtnzlml) [inline android/.project]: What are these files?!

File: android/build.gradle [hunk 2]
```diff
@@ -2,10 +2,11 @@
 
 buildscript {
     repositories {
+        google()// this is the magic line which fixes build failure on yarn android
```
Comment (mrtnzlml) [inline android/build.gradle]: Can you please add a better explanation? I don't like the fact that we have to use magic here but we don't know what it actually means.

**Comment by tbergq** [comment]
I have misread the documentation. It says `The Google Play images are available starting with Android Nougat (API 24)` . So there can be newest API Version. The emulator image though needs to support google play

skjermbilde 2018-02-14 kl 14 02 26

I will add this information to the readme

File: android/.project (hunk 1 above)
Comment (tbergq) [reply android/.project]: I checked with our android guy, and it seems this should be in the `.gitignore` file. I will add it.

**Comment by mrtnzlml** [comment]
This is not fix at all. I managed to make it work even without this fix... 🙄

**Pull request closed without merging**